### PR TITLE
Feature/circle 2

### PR DESCRIPTION
**Pull request by dan-gamble**
This is linked with https://github.com/onespacemedia/server-management/pull/65

### CODE REVIEW

File: {{cookiecutter.repo_name}}/gunicorn_start [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+DJANGO_WSGI_MODULE={{ cookiecutter.package_name }}.wsgi
+
+echo "Starting $NAME as `whoami`"
+
+# Activate the virtual environment.
+cd $DJANGODIR
+source .venv/bin/activate
+
+export DJANGO_SETTINGS_MODULE={{ cookiecutter.package_name }}.settings.production
+export DB_USER=fluidic_analytics
```
Comment (danielsamuels) [inline {{cookiecutter.repo_name}}/gunicorn_start]: Needs to be a variable.

File: {{cookiecutter.repo_name}}/manage.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python
 import os
+import pwd
 import sys
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{cookiecutter.package_name}}.settings.local")
+    os.environ.setdefault("DB_USER", pwd.getpwuid(os.getuid()).pw_name)
+    os.environ.setdefault("DB_NAME", "fluidic_analytics")
```
Comment (danielsamuels) [inline {{cookiecutter.repo_name}}/manage.py]: Variable

File: {{cookiecutter.repo_name}}/manage.py [hunk 3]
```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python
 import os
+import pwd
 import sys
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{cookiecutter.package_name}}.settings.local")
+    os.environ.setdefault("DB_USER", pwd.getpwuid(os.getuid()).pw_name)
```
Comment (danielsamuels) [inline {{cookiecutter.repo_name}}/manage.py:8]: What does this do?

**Review by danielsamuels — CHANGES_REQUESTED** [review]

File: {{cookiecutter.repo_name}}/manage.py (hunk 3 above)
Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/manage.py:8]: This is copied from what was previously in `local.py`. It will get the os user.

Comment (danielsamuels) [reply {{cookiecutter.repo_name}}/manage.py:8]: Oh, fair enough.

**Review by danielsamuels — APPROVED** [review]

**Pull request merged**